flocker/ca/_ca.py

```diff
@@ -619,11 +619,19 @@ def initialize(cls, path, name, begin=None):
             subject and issuer identities of the generated root certificate.
         :param datetime begin: The datetime from which the generated
             certificate should be valid.
+        :param UUID cluster_id: The unique identifier of the cluster for which
+            to generate the key and certificate.  If not given, a random
+            identifier will be generated.
 
         :return RootCredential: Initialized certificate authority.
         """
-        dn = DistinguishedName(commonName=name,
-                               organizationalUnitName=bytes(uuid4()))
+        if cluster_id is None:
+            cluster_id = uuid4()
```

Comment:
Wondering if make_cluster_id could be useful here at some in future? To stick Flocker version?